recipes/r-kaggler/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.0.0.9000" %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-kaggler
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - https://github.com/mkearney/kaggler/archive/530b3af593d44b3e28400b8f581e55a6e09b027a.tar.gz
```

Comment:
We generally want to work downstream of CRAN, as it typically indicates a baseline maturity for a package.